README.md

```diff
@@ -115,6 +115,18 @@ Executing tests that do not pattern match "System"
 npm test -- --ver=alpha --invert --grep="System"
 ```
 
+## Test with core
+
+```bash
+CORE_PATH=*path to core* npm run core
+```
+
+or without specifying the core path (core will be cloned from GH into parent folder):
+
+```bash
+npm run core
```

Comment:
It would be good to provide an example specifying a local copy of the core too

Reply:
https://github.com/HadoukenIO/js-adapter/pull/152/commits/7109bf190f3cfb892d49bd31d879ea3d1e0c0d85